checks/mongo/mongo.go

```diff
@@ -0,0 +1,21 @@
+package mongo
+
+import (
+	"context"
+	"fmt"
+
+	"go.mongodb.org/mongo-driver/mongo"
+	"go.mongodb.org/mongo-driver/mongo/readpref"
+)
+
+// New creates a new MongoDB health check function.
+// It is expects you to provide an already established MongoDB connection.
+func New(client *mongo.Client, rp *readpref.ReadPref) func(ctx context.Context) error {
```

Comment:
Can we maybe use a connection string here instead? I would like to avoid that users need to familiarize with how to use the library/driver we use internally. [This repository](https://github.com/hellofresh/health-go/blob/master/checks/mongo/check.go) has an example.

Reply:
ok, updated.